CMS/translations.py

```diff
@@ -682,8 +682,8 @@
         'cy': 'Y dudalen nesaf'
     },
     'no_courses_found': {
-        'en': 'No courses found, remove or change some filters',
-        'cy': 'Ni chafwyd hyd i gyrsiau dileu newid rhai hidlwyr.'
+        'en': 'No courses found - remove or change some filters',
+        'cy': 'Ni chafwyd hyd i unrhyw cyrsiau - dileu neu newid rhai hidlwyr'
```

Comment:
Just a note to say that these are _my_ interpretations and not necessarily the same as the customer.